modules/dokkatoo-plugin/src/main/kotlin/dokka/plugins/DokkaVersioningPluginParameters.kt

```diff
@@ -83,10 +83,16 @@ constructor(
   @get:Optional
   abstract val renderVersionsNavigationOnAllPages: Property<Boolean>
 
+  init {
+    versionsOrdering.convention(null)
```

Comment:
Fun times. See https://github.com/gradle/gradle/issues/12222

Reply:
I think that having either an empty or null versionsOrdering list is nonsensical, since this will exclude all versions, and so both null/empty values should be excluded from the plugin config (and so it will default to SemVer ordering).

It simplifies the logic a bit and makes https://github.com/gradle/gradle/issues/12222 less relevant.

WDYT?

Reply:
Sounds good to me 👍